O21.Game/Localization/Help/english.md

```diff
@@ -0,0 +1,10 @@
+# U-95
+Programming: Anton Butsev
+Graphics: Evgeniy Harkevich
+Music and sound design: Alexander Chistiakov
+---
+Welcome to the underwater world!
+You are a captain of ther supermodern submarine U-95.
+Your task is not to get caught by terrible monsters. Apart from monsters the underwater expanses contain lots of useful items: diskettes, light bulbs, compasses, etc. Although the most useful one is the lifeline.
+It refills your oxygen supplies or provides sumbarine with multiple usefule equipments.
+Submarine is played by arrow controls or via the mous and buttons on the screen.
```

Comment:
```suggestion
Programming: Anton Butsev
Graphics: Evgeniy Kharkevich
Music and sound design: Alexander Chistiakov
---
Welcome to the underwater world !
You are a captain of a supermodern submarine U-95.
Your task is to avoid getting caught by terrible monsters. Apart from the monsters, the underwater realm contain lots of useful items: diskettes, light bulbs, compasses, etc. Though the most useful one is a lifebuoy:
[TODO Image here]
It allows to refill your oxygen supply or augment the sumbarine with all sorts of useful gadgets.
The submarine is controlled by the arrow keys or via the mouse and buttons on the game screen:
[TODO Image here]
```